Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4434,67 +4433,67 @@
     ]
   },
   {
-    "Input": "Ya terminé todos los trabajos hace más de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 semanas antes de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "hace más de 2 semanas",
+        "Text": "durante más de 2 semanas antes de hoy",
         "Type": "daterange",
-        "Start": 30,
-        "Length": 21
+        "Start": 0,
+        "Length": 37
       }
     ]
   },
   {
-    "Input": "Volveré dentro de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de 2 semanas a partir de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "dentro de 2 semanas",
+        "Text": "dentro de 2 semanas a partir de hoy",
         "Type": "daterange",
-        "Start": 8,
-        "Length": 19
+        "Start": 0,
+        "Length": 35
       }
     ]
   },
   {
-    "Input": "Volveré dentro de menos de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de menos de 2 semanas a partir de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "dentro de menos de 2 semanas",
+        "Text": "dentro de menos de 2 semanas a partir de hoy",
         "Type": "daterange",
-        "Start": 8,
-        "Length": 28
+        "Start": 0,
+        "Length": 44
       }
     ]
   },
   {
-    "Input": "Esta tarea debió ser terminada más de tres días antes",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 días antes de ayer",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "más de tres días antes",
+        "Text": "durante más de 2 días antes de ayer",
         "Type": "daterange",
-        "Start": 31,
-        "Length": 22
+        "Start": 0,
+        "Length": 35
       }
     ]
   },
   {
-    "Input": "Esta tarea debe ser terminada en menos de 4 días",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de menos de 3 días a partir de mañana",
```

Comment:
"Dentro de menos" is not exactly the same as "en menos de". The first form sounds strange to me.
But I'm not a native Spanish speaker. Also, we need to support both Spanish from Spain and from all Latin America. Before removing some form, we need to try to confirm if it's not common in specific countries.

Reply:
Get it, the "dentro de menos de 3 días a partir de mañana" is translated from English DatePeriod cases "This task will be done less than 3 days after tomorrow", I will support both two cases in next pr